content/en/blog/news/Hertz_Open_Source_announcement/index.md

```diff
@@ -0,0 +1,66 @@
+---
+date: 2022-06-21
+title: "Hertz, an Ultra Large Scale Enterprise-Level Microservice HTTP Framework, is Now Officially Open Source!"
+linkTitle: "Hertz, an Ultra Large Scale Enterprise-Level Microservice HTTP Framework, is Now Officially Open Source!"
+keywords: ["CloudWeGo", "http framework", "large scale high performance", "Hertz", "ByteDance Open Source", "open source"]
+description: "This article introduces Hertz, the official open source ultra-large-scale enterprise-level microservice HTTP framework developed by ByteDance"
+author: <a href="https://github.com/cloudwego" target="_blank">CloudWeGo Team</a>
+---
+Today, after more than a year of internal use and iteration at ByteDance, **[Hertz](https://github.com/cloudwego/hertz)**, a **high-performance enterprise-level HTTP framework**, is officially open-sourced on [CloudWeGo](https://github.com/cloudwego)! Hertz has become the largest HTTP framework within ByteDance, with more than **10,000** online services and a peak QPS (Query Per Second) of more than **40 million**. It has the characteristics of high usability, easy expansion, and low latency. For the ByteDance service framework team and CloudWeGo, [Hertz](https://github.com/cloudwego/hertz) will not only be an open source project, but also a real ultra-large-scale enterprise-level practice.
+
+Project website: https://github.com/cloudwego/hertz
```

Comment:
website -> github

Reply:
Did you mean change from https://github.com/cloudwego to https://www.cloudwego.io/

Reply:
Fixed in the new commit.